src/shims/generic_win_stubs.c

```diff
@@ -1,3 +1,94 @@
+#include "internal.h"
+
+/*
+ * This file contains stubbed out functions we are using during
+ * the initial Windows port.  When the port is complete, this file
+ * should be empty (and thus removed).
+ */
+
+void
+_dispatch_runloop_queue_dispose(dispatch_queue_t dq DISPATCH_UNUSED,
+		bool *allow_free DISPATCH_UNUSED)
+{
+	WIN_PORT_ERROR();
+}
+
+void
+_dispatch_runloop_queue_xref_dispose(dispatch_queue_t dq DISPATCH_UNUSED)
+{
+	WIN_PORT_ERROR();
+}
+
+void _dispatch_event_loop_poke(dispatch_wlh_t wlh DISPATCH_UNUSED,
+		uint64_t dq_state DISPATCH_UNUSED, uint32_t flags DISPATCH_UNUSED)
+{
+	WIN_PORT_ERROR();
+}
+
+void
+_dispatch_event_loop_wake_owner(dispatch_sync_context_t dsc DISPATCH_UNUSED,
+		dispatch_wlh_t wlh DISPATCH_UNUSED, uint64_t old_state DISPATCH_UNUSED,
+		uint64_t new_state DISPATCH_UNUSED)
+{
+	WIN_PORT_ERROR();
+}
+
+void
+_dispatch_event_loop_wait_for_ownership(
```

Comment:
for this family of functions, you should instead start a: src/event/event_windows.c and copy the functions from event_epoll.c that do nothing and or assert.

they will likely not have an implementation on windows for a while in the first place.